docs/security-spec.md

```diff
@@ -0,0 +1,1080 @@
+﻿# Security Specification
+
+#### 4. Security
+   - 4. Security
```

Comment:
See the above discussion. Team needs to decide if the protocol spec is good as-is, or if it should be split into multiple documents, such as Security. 4. ... (This was the start of doing that, as an iterative process, but now there are objections ... ) The TOC linking will be resolved by the new Sphinx/Python/ReadtheDocs build, which is coming soon and being worked in another issue.